tests/integration/test_render.py

```diff
@@ -972,11 +972,11 @@ def render_content(tab):
         )[0].click()
 
         graph_1_expected_clickdata = {
-            "points": [{"curveNumber": 0, "pointNumber": 1, "pointIndex": 1, "x": 2, "y": 10}]
+            "points": [{"curveNumber": 0, "pointNumber": 1, "pointIndex": 1, "x": 2, "y": 10, "label": 2, "value": 10}]
         }
 
         graph_2_expected_clickdata = {
-            "points": [{"curveNumber": 0, "pointNumber": 1, "pointIndex": 1, "x": 3, "y": 10}]
+            "points": [{"curveNumber": 0, "pointNumber": 1, "pointIndex": 1, "x": 3, "y": 10, "label": 3, "value": 10}]
```

Comment:
@alexcjohnson Getting the same error you have in #1026 -- there seems to be additional label/value props

Reply:
Added https://github.com/plotly/dash/issues/1031 for follow up